src/wifi_manager.py

```
@@ -1401,7 +1401,7 @@ def connect_to_network(self, ssid: str, password: str) -> Tuple[bool, str]:
                     # Last resort: enable AP mode
                     try:
                         self.enable_ap_mode()
-                    except Exception:
+                    except Exception:  # nosec B110 - last-resort recovery; if AP enable fails there's nothing left to try
                         pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)